code/game/machinery/launch_pad.dm

```diff
@@ -71,6 +71,15 @@
 
 	return ..()
 
+/obj/machinery/launchpad/attack_ghost(mob/dead/observer/ghost)
+	. = ..()
+	if(.)
+		return
+	var/target_x = x + x_offset
+	var/target_y = y + y_offset
+	var/turf/target = locate(target_x, target_y, z)
+	ghost.forceMove(get_turf(target))
```

Comment:
Is the get_turf needed? You're getting the turf of a turf.

Reply:
Yeah, good point

Reply:
Do this please.